src/m_misc.c

```diff
@@ -2594,6 +2594,13 @@ default_t defaults[] = {
     "1 to disable doubled card and skull key display on status bar"
   },
 
+  {
+    "hud_blink_keys",
+    (config_t *) &hud_blink_keys, NULL,
+    {1}, {0,1}, number, ss_none, wad_yes,
+    "1 to make missing keys blink when trying to open locked doors"
```

Comment:
Doors aren't the only specials that may require a key, so this description should be changed. Maybe "key-requiring specials/actions"? Once again, I'm open to suggestions.

Reply:
"1 to make missing keys blink when trying to trigger linedef actions" maybe. The "missing keys" part already suggests "key-requiring actions".

Reply:
> "1 to make missing keys blink when trying to trigger linedef actions" maybe.

Since no other suggestions were made, I applied this in 38c2de3f1fabb9ef8cde8fe6c680860e0968cd4d.